ObjectTracker:
{
	filename_camera: "camera_calibration.yml",
	size_marker: 0.02 
}
